packages/multi-test/src/test_helpers.rs

```diff
@@ -3,17 +3,24 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
-use cosmwasm_std::{
-    to_binary, BankMsg, Binary, Coin, Deps, DepsMut, Empty, Env, Event, MessageInfo, Reply,
-    Response, StdError, SubMsg,
-};
-use cw_storage_plus::{Item, Map, U64Key};
+use cosmwasm_std::{Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdError};
+use cw_storage_plus::Item;
 
 use crate::contracts::{Contract, ContractWrapper};
 
+pub mod contracts;
+
 #[derive(Debug, Clone, Serialize, Deserialize, Default)]
 pub struct EmptyMsg {}
 
+/// This is just a demo place so we can test custom message handling
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, PartialEq)]
+#[serde(rename = "snake_case")]
+pub enum CustomMsg {
+    SetName { name: String },
+    SetAge { age: u32 },
+}
+
 fn instantiate_error(
```

Comment:
Is there a reason you left error top level and didn't pull it into a separate file like payout and reflect?

I know it is smaller, but was there another reason?
(I like your reorg)

Reply:
I decided that those functions are just not part of specific contracts, they are generic to be used in other contracts. I actually used `query_error` this way in my new contract (not yet here).

Reply:
Forget it - I just missed there is whole contract for errors. I am blind.